playbooks/roles/credentials/defaults/main.yml

```diff
@@ -132,7 +135,19 @@ CREDENTIALS_CORS_ORIGIN_WHITELIST_DEFAULT:
 CREDENTIALS_CORS_ORIGIN_WHITELIST_EXTRA: []
 CREDENTIALS_CORS_ORIGIN_WHITELIST: '{{ CREDENTIALS_CORS_ORIGIN_WHITELIST_DEFAULT + CREDENTIALS_CORS_ORIGIN_WHITELIST_EXTRA }}'
 
+CREDENTIALS_GIT_PROTOCOL: '{{ COMMON_GIT_PROTOCOL }}'
+CREDENTIALS_GIT_MIRROR: '{{ COMMON_GIT_MIRROR }}'
+CREDENTIALS_GIT_PATH: '{{ COMMON_GIT_PATH }}'
 CREDENTIALS_VERSION: 'master'
+CREDENTIALS_GIT_IDENTITY: null
+CREDENTIALS_REPOS:
```

Comment:
I don't understand why this had disappeared between Ginkgo and Hawthorn Configuration branches

Reply:
Yup, it was part of a big refactor. Those values makes no sense for edX. So they've removed it. We want it though:

 - https://github.com/edx/configuration/commit/b6d45554f05fedd8d331fa9d8515095f68c20aeb#diff-240da08108c4d3cce14b2abd689b924eL13